.github/workflows/lint_javascript.yml

```diff
@@ -0,0 +1,20 @@
+name: Test javascript
+on:
+  pull_request:
+    paths:
+    - 'javascript/**'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
Hi @AlexLloyd0 please consider using a "Cancel previous" workflow as your first step in all actions:

```yaml
steps:
  - name: Cancel previous
    uses: styfle/cancel-workflow-action@0.8.0
    with:
      access_token: ${{ github.token }}
```

By default, GitHub Actions will always build for every commit. If you're working on a PR, that means each push or rebase will trigger a full run and will not stop automatically. 4 jobs on a PR with 10 commits == 40 jobs (we only have 180 for the entire google org).

Reply:
Done - thanks for the feedback!